config/swagger.yaml

```diff
@@ -2245,7 +2247,7 @@ definitions:
     type: object
     properties:
       oracle_pubkey:
```

Comment:
This property name may be misleading (pubkey while it can be name)?

Reply:
it is allowed to be an encoded pubkey or a name string